packages/deeplinks/observability/locators/dataset_quality.ts

```diff
@@ -23,11 +23,23 @@ type TimeRangeConfig = {
   refresh: RefreshInterval;
 };
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+type DatasetConfig = {
+  rawName: string;
+  type: string;
+  name: string;
+  namespace: string;
+  title: string;
```

Comment:
Why is title required here? 
at least other properties could be derived from the rawName somehow. In any case I'd expect something more close to
```
flyout? {
    dataset: string;
}
```
and the locator at the end being the responsible of transforming that rawName into the object we need to pass to the state. 
Is it something I'm missing here?

Reply:
It's a good catch, looks like i added this before i did the other PR refactoring the integration. I will change this. Let me see the best way to do this. Your idea to pass a dataset as a string, i will evaluate what happens with the URL when by default the Flyout is opened.

Reply:
As discussed offline, we will just remove the title for the time being and resolve the rest while working in the context of https://github.com/elastic/kibana/issues/184572